src/lib/pages/account-details/components/AccountHeader.tsx

```diff
@@ -147,6 +211,25 @@ export const AccountHeader = observer(
                 />
               </Flex>
             )}
+            {accountLocalInfo?.name && (
```

Comment:
this condition looks incorrect. what is the intention of this condition?

Reply:
If user also have account local name, we will show initia username below as well

Reply:
okie

Reply:
add condition `&& data` here so it won't render if there is no username